test/hotspot/jtreg/compiler/c2/gvn/TestXorInt.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
We have `compiler/c2/irTests/XorINodeIdealizationTest.java`, is adding a separate test file necessary?